lib/student.rb

```diff
@@ -1 +1,19 @@
+class Student
+  attr_reader :name, 
+              :age, 
+              :scores
 
+  def initialize(info)
+    @name = info[:name]
+    @age = info[:age]
+    @scores = []
+  end
+
+  def log_score(score)
+    scores << score
+  end
+
+  def grade
+    scores.sum.fdiv(2)
+  end
```

Comment:
This should be dynamic. What if a student has logged more than 2 scores?
```suggestion
  def grade
    scores.sum.fdiv(@scores.count)
  end
```